xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/DefaultLiveDataConfigurationResolver.java

```diff
@@ -187,6 +187,13 @@ private void addMissingPropertyDescriptors(LiveDataConfiguration config, List<St
                 .map(this::getDefaultPropertyDescriptor).collect(Collectors.toList());
         if (!missingDescriptors.isEmpty()) {
             propertyDescriptors = new ArrayList<>(propertyDescriptors);
+            // By default, a property that is declared in the macro configuration but is not a known XClass property
+            // and does not have a explicit 'editable' configuration is defined as not editable.
```

Comment:
I would make this about XClass properties. I'd simply say that without any other information (property descriptor) it's safer to assume that properties with missing descriptors are not editable, not sortable and not filterable, by default.